### ISSUE TITLE
commenting false positives in JSON files

### ISSUE THREAD

**Issue by seefood** [Yelp/detect-secrets#264]
If I may suggest, there needs to be a way to comment on JSON lines. for instance, I tried to add a remark to a false positive line, but no dumb luck. Is there a way to do it inline without excluding the entire file forever and missing out on future problems?

(example is an ERB template)

```
<% if @rto_redis.key?('password') %>
  "redis_rto": {
    "host": "<%= @rto_redis['host'] %>",
    "port": "<%= @rto_redis['port'] %>",
    "password": "<%= @rto_redis['password'] %>", "_comment": "/* pragma: allowlist secret */",
    "database": <%= @rto_redis['database'] %>,
    "ssl": true
  },
<% end %>
```

**Comment by domanchi** [comment]
JSON comments are definitely a PITA. That's one of the reasons why we use YAML files internally -- because commenting configuration files is definitely a plus! If you can figure out a non hacky way to perform JSON comments, let us know, and maybe we'll build out support for that too.

I'm confused by your example though -- you mention commenting JSON lines, but show an ERB template? How are they related? If you are using it in an ERB template, then that should support comments, right?

**Comment by seefood** [comment]
True, this specific case is an ERB, but I also have real JSONs to sweep, and I was hoping to find some sort of a standard solution for the general case.
